src/main/java/com/google/sps/data/Event.java

```diff
@@ -57,10 +59,11 @@ public double getPrice() {
         return this.price;
     }
 
-    public long getDate(){
-        /**
-         * 'date' is returned with epoch format (ex. 1617136856)
+    public String getDate(){
```

Comment:
You guys think this getter should be called `getEpoch`? it just seems not descriptive.

Reply:
I think `getDate()` is fine, since that's what you're actually returning